dot/state/mock_counter_test.go

```diff
@@ -21,7 +21,6 @@ import (
 type MockCounter struct {
 	ctrl     *gomock.Controller
 	recorder *MockCounterMockRecorder
-	isgomock struct{}
```

Comment:
When I generate a mock, It adds this line.
And here it's removed.

Why does it behave differently? @EclesioMeloJunior @timwu20

Reply:
I am using mockgen version 0.4.0, also the CI is using this version. What version from mockgen are you using? @axaysagathiya

Reply:
I will update it then

Reply:
https://github.com/ChainSafe/gossamer/pull/4589